tests/test_rmsd_align.py

```diff
@@ -0,0 +1,38 @@
+# test the custom_op for rmsd alignment
+
+from timemachine.lib import custom_ops
+
+from timemachine.potentials import rmsd
+import numpy as np
+from testsystems import relative
+from scipy.stats import special_ortho_group
+
+
+def test_rmsd_align():
+
+    N = 25
+
+    for _ in range(100):
+
+        x1 = np.random.rand(N,3)
+        random_t = np.random.rand(3)
+        random_R = special_ortho_group.rvs(3)
+        x2 = x1@random_R + random_t
```

Comment:
Question: should this also exercise the case where `is_reflection` is true? https://github.com/proteneer/timemachine/blob/1ba4b20ea3efc4631f77cb637661ba2c78bb47eb/timemachine/cpp/src/rmsd_align.cpp#L51-L54

(If so, changing the test to use `ortho_group` instead of `special_ortho_group` might do the trick.)

Reply:
I'd thought of doing this initially, but I wanted to initially write a version where I can assert that the optimally aligned RMSD is zero. In the case of an improper rotation projected from O(3) into SO(3), the RMSD would no longer be zero. But you're right that we should still add a test for this, at least to ensure that the RMSD is smaller and that the python/cpp results are consistent.

Reply:
PS thanks a lot for this comment! this actually helped me catch a bug in how I implemented the reflection case.